db/src/Pos/DB/Block/Logic/Internal.hs

```diff
@@ -203,9 +207,15 @@ applyBlocksDbUnsafeDo genesisConfig scb blunds pModifier = do
     usBatch <- SomeBatchOp <$> usApplyBlocks genesisConfig (map toUpdateBlock blocks) pModifier
     delegateBatch <- SomeBatchOp <$> dlgApplyBlocks (map toDlgBlund blunds)
     txpBatch <- tgsApplyBlocks $ map toTxpBlund blunds
-    sscBatch <- SomeBatchOp <$>
-        -- TODO: pass not only 'Nothing'
-        sscApplyBlocks genesisConfig (map toSscBlock blocks) Nothing
+    era <- getConsensusEra
+    sscBatch <- case era of
+        Original -> do
+            -- TODO: pass not only 'Nothing'
```

Comment:
Should this TODO be acted on?

Reply:
Actually, I hadn't looked into it. It looks like this comment was there prior to these changes. I'll take a look to see what the concern is.

Reply:
Oh, didn't realize it was just moved. Probably worth a brief looking-into.

Reply:
I feel like we're better off leaving it since it has been working. We won't be using this functionality when we transition to `OBFT` anyways.